cumulusci/cli/cci.py

```diff
@@ -1449,83 +1442,178 @@ def task_info(runtime, task_name):
     click.echo(rst2ansi(doc))
 
 
-@task.command(name="run", help="Runs a task")
-@click.argument("task_name")
-@click.option(
-    "--org",
-    help="Specify the target org.  By default, runs against the current default org",
-)
-@click.option(
-    "-o",
-    nargs=2,
-    multiple=True,
-    help="Pass task specific options for the task as '-o option value'.  You can specify more than one option by using -o more than once.",
-)
-@click.option(
-    "--debug", is_flag=True, help="Drops into pdb, the Python debugger, on an exception"
-)
-@click.option(
-    "--debug-before",
-    is_flag=True,
-    help="Drops into the Python debugger right before task start.",
-)
-@click.option(
-    "--debug-after",
-    is_flag=True,
-    help="Drops into the Python debugger at task completion.",
-)
-@click.option(
-    "--no-prompt",
-    is_flag=True,
-    help="Disables all prompts.  Set for non-interactive mode use such as calling from scripts or CI systems",
-)
-@pass_runtime(require_keychain=True)
-def task_run(runtime, task_name, org, o, debug, debug_before, debug_after, no_prompt):
+class RunTaskCommand(click.MultiCommand):
+    # options hat are not task specific
+    global_options = {
+        "org": {
+            "help": "Specify the target org. By default, runs against the current default org.",
+            "is_flag": True,
```

Comment:
I guess this is overwritten on line 1604, but it's not a boolean flag.

Reply:
I'll remove it.